### PR TITLE
Hide iPXE on APU7

### PR DESCRIPTION
**Pull request by mkopec**
Signed-off-by: Michał Kopeć <michal.kopec@3mdeb.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -92,6 +92,10 @@ else
 	CFLAGS += -DFCH_YANGTZEE
 endif
 
+ifeq ($(IPXE),y)
```
Comment (miczyg1) [inline Makefile:95]: Not needed. Please take a look at: https://github.com/pcengines/sortbootorder/pull/81/files#diff-4062b3f978b19a82ecef89231f80c0e646b86174e6587d0c82ff03277838416dR156
Here the mainboard name is retrieved. Just strcmp it to apu7 and make a variable called `no_pxe` or something like that and simply do not handle any options regarding PXE.

Comment (mkopec) [reply Makefile]: Done as per suggestion.

**Comment by miczyg1** [comment]
One more things, the iPXE wil lbe also listed in the boot order here: https://github.com/pcengines/sortbootorder/pull/81/files#diff-4062b3f978b19a82ecef89231f80c0e646b86174e6587d0c82ff03277838416dR458

We do not want that. We have to filter the iPXE and do not print it here.

**Comment by mkopec** [comment]
@miczyg1 Added boot device list filtering so that iPXE is hidden on apu7

**Review by miczyg1 — APPROVED** [review]

**Pull request merged**